shell/platform/darwin/ios/framework/Source/FlutterAppDelegate.mm

```diff
@@ -3,7 +3,28 @@
 // found in the LICENSE file.
 
 #include "flutter/shell/platform/darwin/ios/framework/Headers/FlutterAppDelegate.h"
+#include "flutter/shell/platform/darwin/ios/framework/Headers/FlutterViewController.h"
 
 @implementation FlutterAppDelegate
 
```

Comment:
We should add a comment here saying that you don't have to inherit from this class and that if you don't then you can copy-paste this code to yours to make sure it's working, etc.